projects/backend/controllers/editions.ts

```diff
@@ -12,7 +13,36 @@ const editionsList: EditionsList = {
     trainingEditions: [],
 }
 
-export const editionsController = (req: Request, res: Response) => {
+export const editionsControllerGet = (req: Request, res: Response) => {
     res.setHeader('Content-Type', 'application/json')
     res.send(JSON.stringify(editionsList))
 }
+
+export const editionsControllerPost = async (req: Request, res: Response) => {
+    try {
+        // TODO: doing simple validation now but would be good to enhance this further
```

Comment:
I think it might be good to at least add a basic function that checks that there is a property called regionalEditions and that it has at least 3 members, we can leave full validation for later.